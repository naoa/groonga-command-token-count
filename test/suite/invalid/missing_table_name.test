register commands/token_count

table_create Entries TABLE_NO_KEY
column_create Entries body COLUMN_SCALAR ShortText

table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram --normalizer NormalizerAuto
column_create Terms document_index COLUMN_INDEX|WITH_POSITION Entries body

load --table Entries
[
{"body": "今日は雨。今日は調子が良い。今日は雨"},
{"body": "今日は雨。今日は調子が悪い"}
]

token_count --token_size -1 --use_ctx_output 1
