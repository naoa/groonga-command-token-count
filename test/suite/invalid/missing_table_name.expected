register commands/token_count
[[0,0.0,0.0],true]
table_create Entries TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Entries body COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram --normalizer NormalizerAuto
[[0,0.0,0.0],true]
column_create Terms document_index COLUMN_INDEX|WITH_POSITION Entries body
[[0,0.0,0.0],true]
load --table Entries
[
{"body": "今日は雨。今日は調子が良い。今日は雨"},
{"body": "今日は雨。今日は調子が悪い"}
]
[[0,0.0,0.0],2]
token_count --token_size -1 --use_ctx_output 1
[[[-22,0.0,0.0],"[token_count] table name is missing"]]
#|e| [token_count] table name is missing
